eng/testing/performance/android_scenarios.proj

```diff
@@ -77,6 +89,12 @@
       <Command>$(Python) test.py devicestartup --device-type android --package-path pub\MauiAndroidDefault.apk --package-name com.companyname.mauitesting --scenario-name &quot;%(Identity)&quot;</Command>
       <PostCommands>$(Python) post.py</PostCommands>
     </HelixWorkItem>
+    <HelixWorkItem Include="Device Startup - Android Maui Blazor DefaultTesting">
+      <PayloadDirectory>$(WorkItemDirectory)</PayloadDirectory>
+      <PreCommands>echo on;set XHARNESSPATH=$(XharnessPath);cd $(ScenarioDirectory)mauiandroid;copy %HELIX_CORRELATION_PAYLOAD%\MauiBlazorAndroidDefault.apk .;$(Python) pre.py --apk-name MauiBlazorAndroidDefault.apk</PreCommands>
+      <Command>$(Python) test.py devicestartup --device-type android --package-path pub\MauiBlazorAndroidDefault.apk --package-name com.companyname.mauiblazortesting --scenario-name &quot;%(Identity)&quot;</Command>
+      <PostCommands>$(Python) post.py</PostCommands>
+    </HelixWorkItem>
```

Comment:
@LoopedBard3 how should this be integrated with the time to first draw reading from LogCat?

https://github.com/dotnet/performance/pull/2345

Reply:
@TanayParikh The only additional things you should need is adding '--use-fully-drawn-time' and maybe '--fully-drawn-extra-delay' as arguments to the test.py command. '--use-fully-drawn-time' will use the fully drawn time rather than the basic app container startup time. '--fully-drawn-extra-delay <seconds>' will add a longer delay from startup to take into account the time needed to fully draw the app. The default time is 3 seconds so you don't need this arg if the fully drawn - startup time is less than 3 (2 to be safe) seconds.

Reply:
Thanks @LoopedBard3, updated with the two args. Local testing shows a time of 2.913s, so I definitely think the delay arg is warranted. I noticed the CI failed last time, is that expected, because https://github.com/dotnet/performance/pull/2386 isn't merged?

Reply:
Yes, we should merge the https://github.com/dotnet/performance/pull/2386 first.